Cargo.toml

```diff
@@ -24,4 +26,6 @@ log = "0.4.0"
 lazy_static = "1.4.0"
 colored = "2.0.0"
 probe-rs = { version = "0.9.0", git = "https://github.com/probe-rs/probe-rs" }
-probe-rs-cli-util = { version = "0.9.0", git = "https://github.com/probe-rs/probe-rs" }
+probe-rs-cli-util = { version = "0.9.0", git = "https://github.com/probe-rs/probe-rs", default-features = false }
+git-version = "0.3.4"
+backtrace = "0.3"
```

Comment:
For what is this crate needed?

Reply:
Which one? GH is not very precise about this :/

Reply:
The `backtrace` crate.